pkg/transfer/local/transfer.go

```diff
@@ -150,3 +150,40 @@ func (ts *localTransferService) withLease(ctx context.Context, opts ...leases.Op
 		return ls.Delete(ctx, l)
 	}, nil
 }
+
+type TransferConfig struct {
+	// MaxConcurrentDownloads is the max concurrent content downloads for pull.
+	MaxConcurrentDownloads int `toml:"max_concurrent_downloads"`
```

Comment:
Not related to this pr. I just see many issues saying that the image pulling brings more io pressure. So I am wondering about how to add a global limitation here. maybe introduces rate-limiter var?

Reply:
We do have a global limiter defined [here](https://github.com/containerd/containerd/pull/8062/files#diff-b95013e1f6d0e346d260a99d79c838358675ad087c84b082dc35024ce9b56ed9R43) which [initialized](https://github.com/containerd/containerd/pull/8062/files#diff-b95013e1f6d0e346d260a99d79c838358675ad087c84b082dc35024ce9b56ed9R55) using this `MaxConcurrentDownloads `

Reply:
Thanks! I misunderstand this part when I review it. It will change the existing CRI-plugin config definition right now.
Currently, the CRI-plugin limits the concurrent pulling numbers for each pull. It confuses that users from docker.
It looks good to me if the CRI-plugin config definition can align with transfer service. I was thinking about adding a global rate-limit for CRI-plugin.